addons/web/static/src/core/dropdown/dropdown_item.xml

```diff
@@ -2,11 +2,11 @@
 <templates xml:space="preserve">
 
   <t t-name="web.DropdownItem" owl="1">
-    <li class="o_dropdown_item" t-on-click="onClick" t-att-title="props.title" t-att-data-hotkey="props.hotkey">
+    <a class="o_dropdown_item" t-on-click="onClick" t-att-title="props.title" t-att-data-hotkey="props.hotkey" t-att-href="props.href">
```

Comment:
always an <a> even if no href?

Reply:
when undefined, [href] attribute is not present in the dom.
is it ok or you suggest something else ?

Reply:
Simple workaround: always set an [href], even if empty